fingertip/plugins/self_test/script.py

```diff
@@ -0,0 +1,15 @@
+# Licensed under GNU General Public License v3 or later, see COPYING.
+# Copyright (c) 2019 Red Hat, Inc., see CONTRIBUTORS.
+
+import fingertip
+import fingertip.util.temp
+
+@fingertip.transient
+def main(m):
+    scriptname = fingertip.util.temp.disappearing_file()
+    print(f"Scriptname: {scriptname}")
```

Comment:
please use m.log.debug, with all that stuff that I do with logging, prints should be broken by now =D